domain/src/unit-test/java/uk/gov/moj/sdt/domain/IndividualRequestTest.java

```diff
@@ -64,116 +72,118 @@ public void setUp() {
      * Tests that forwarding attempts is incremented correctly.
      */
     @Test
-    public void testIncrementForwardingAttempts() {
+    void testIncrementForwardingAttempts() {
         individualRequest.incrementForwardingAttempts();
 
-        Assert.assertEquals("Status is incorrect", IndividualRequestStatus.FORWARDED.getStatus(),
-                individualRequest.getRequestStatus());
-        Assert.assertEquals("Forwarding attempt count is incorrect", 1, individualRequest.getForwardingAttempts());
-        Assert.assertNotNull("Updated date should be populated", individualRequest.getUpdatedDate());
-
+        assertEquals(IndividualRequestStatus.FORWARDED.getStatus(), individualRequest.getRequestStatus(),
+                STATUS_IS_INCORRECT);
+        assertEquals(1, individualRequest.getForwardingAttempts(),
+                FORWARDING_ATTEMPT_COUNT_IS_INCORRECT);
+        assertNotNull(individualRequest.getUpdatedDate(), UPDATED_DATE_SHOULD_BE_POPULATED);

```

Comment:
These three checks are common to a lot of the tests in this class.  Class could be improved by moving them into a separate method that has expected status and expected forwarding count parameters and calling that from each test.